packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.css

```diff
@@ -0,0 +1,45 @@
+.root {
+    display: grid;
+    padding: 2.5rem 3rem;
+    row-gap: 2rem;
+}
+
+.title {
+    justify-self: center;
+    font-family: var(--venia-global-fontFamily-serif);
+    font-weight: var(--venia-global-fontWeight-bold);
+}
+
+.lineItems {
+    display: grid;
+    grid-gap: 0.75rem;
+    grid-template-columns: 1fr auto;
+    line-height: 1.5rem;
+}
+
+.lineItemLabel {
+    font-weight: var(--venia-global-fontWeight-bold);
+}
+
+.editInformationButton {
+    composes: root_normalPriority from '../Button/button.css';
+    margin: auto;
+}
+
+@media (min-width: 961px) {
+    .cardArea {
```

Comment:
This is an interesting class name. Can we update it to reflect what's being shown on the page? Maybe `accountDetails` or something?